modules/claims_api/spec/requests/v1/claims_request_spec.rb

```diff
@@ -47,7 +48,7 @@
 
   context 'index' do
     it 'lists all Claims', run_at: 'Tue, 12 Dec 2017 03:09:06 GMT' do
-      with_okta_user(scopes) do |auth_header|
```

Comment:
Really appreciate the adjustment here to decouple from okta.

Drives us towards the ultimate objective from a security standpoint.